components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultAuthenticationRequestHandler.java

```diff
@@ -277,7 +280,7 @@ protected void concludeFlow(HttpServletRequest request, HttpServletResponse resp
                 // TODO add to cache?
                 // store again. when replicate  cache is used. this may be needed.
                 FrameworkUtils.addSessionContextToCache(commonAuthCookie, sessionContext);
```

Comment:
Use sessionContextKey to save session context like follows.

FrameworkUtils.addSessionContextToCache(sessionContextKey, sessionContext);